risc0/zkvm/src/lib.rs

```diff
@@ -23,12 +23,18 @@ extern crate alloc;
 #[cfg(feature = "binfmt")]
 pub mod binfmt;
 mod control_id;
+#[cfg(feature = "prove")]
+mod exec;
```

Comment:
If a user wants to implement a syscall handler in the Executor, wouldn't this need to be pub to access the `risc0_zkvm::exec::io::{Syscall, SyscallContext}`? 

I am currently testing using the new Executor api and hit the missing SYS_RANDOM in Execute mode, so tried to implement it as a custom handler and hit this.

Reply:
Makes sense to me!